src/Illuminate/Database/Eloquent/Relations/MorphToMany.php

```diff
@@ -121,13 +121,14 @@ public function getRelationExistenceQuery(Builder $query, Builder $parentQuery,
     }
 
     /**
-     * Get the pivot models that are currently attached.
+     * Get the pivot models that are currently attached, filtered by related model keys.
      *
-     * @return \Illuminate\Support\Collection<int, TPivotModel>
+     * @param  mixed  $ids
+     * @return \Illuminate\Support\Collection
```

Comment:
Shouldn't this stay
```suggestion
     * @return \Illuminate\Support\Collection<int, TPivotModel>
```
? If there is trouble with static analysis, the [PHPDoc comment in `InteractsWithPivotTable`](https://github.com/laravel/framework/blob/12.x/src/Illuminate/Database/Eloquent/Relations/Concerns/InteractsWithPivotTable.php#L496) should be adjusted accordingly